src/providers/ColorSchemeProvider.tsx

```diff
@@ -43,14 +45,15 @@ export const ColorSchemeProvider: FC = ({ children }) => {
     (newColorScheme: SettingColorSchemeName) => {
       const oldColorScheme = colorSchemeSetting
       setColorSchemeSetting(newColorScheme)
+      setColorMode(newColorScheme)
```

Comment:
There can be some bugs after first installation of the app. When I'm analyzing videos that you provided looks like this can be issue with this.

Maybe you should run function `setColorMode` in `useEffect` function too?

Reply:
I made some little refactor that changed the initialization of a theme a bit. Unless I didn't make any mistake, executing `setColorMode` in `useEffect` is no longer necessary